### PR TITLE
[v0.6.x] Added account commands - WIP

### PR DESCRIPTION
**Pull request by neha-viswanathan**
Ref: #678, #679 

Added CLI implementation of `account` command - WIP

### CODE REVIEW

File: cmd/amp/account.go [hunk 1]
```diff
@@ -40,198 +55,759 @@ var (
 	switchRoleCmd = &cobra.Command{
 		Use:   "switch [ORGANIZATION]",
```
Comment (subfuzion) [inline cmd/amp/account.go]: This shouldn't be an optional parameter. And switch is for ACCOUNT, not ORGANIZATION. You might switch to the account that is the primary account directly associated with your account id (ie, your individual account), or you might switch to an organization account.

File: cmd/amp/account.go [hunk 2]
```diff
@@ -40,198 +55,759 @@ var (
...
 		Short: "Switch primary organization",
+		Long:  `Switch command changes the primary organization of a resource to the specified organization.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return switchRole(AMP, cmd, args)
+		},
 	}
 
 	createOrganizationCmd = &cobra.Command{
-		Use:   "create organization [NAME] [EMAIL]",
+		Use:   "createorg [NAME] [EMAIL]",
```
Comment (subfuzion) [inline cmd/amp/account.go]: I don't like the way these are coming out ... `createorg`, `listteam`, etc. We need to think through a better command-noun-verb-attribute model, but I would think something more like: `amp account create --organization` or `amp account create --type=organization` (default `type=user`)... Let's discuss more.

**Review by subfuzion — CHANGES_REQUESTED** [review]

File: cmd/amp/account.go (hunk 1 above)
Comment (neha-viswanathan) [reply cmd/amp/account.go]: Working on it. Will put it up as a separate PR.

File: cmd/amp/account.go (hunk 2 above)
Comment (neha-viswanathan) [reply cmd/amp/account.go]: I have added the command usage in the doc. Waiting for consensus from the team.

Comment (subfuzion) [reply cmd/amp/account.go]: @neha-viswanathan I've been reviewing and taking a cut at it.

**Pull request closed without merging**